integration-tests/pages/evidencePage.js

```diff
@@ -25,7 +25,9 @@ export default () =>
       cy.get('[name="bodyWornCameraNumbers[1][cameraNum]"]').type('456')
       cy.get('[data-qa-add-another-camera = true]').click()
       cy.get('[name="bodyWornCameraNumbers[2][cameraNum]"]').type('789')
-      cy.get('.add-another-camera > :nth-child(2) > .govuk-button').click()
+      cy.get('.add-another-camera .add-another__remove-button')
```

Comment:
Feels a bit inconsistent here, above we've not tied ourselves to the document structure whereas for the camera buttons we have, could make for a brittle test?

Reply:
I agree, I think this is a major improvement from before which relied on a specific relationship between 3 elements - this also is more descriptive as at least we know which button is being clicked. I've refactored the other page objects to be a bit more explicit in structure but haven't got to this one yet